.github/workflows/extract.yml

```
@@ -0,0 +1,59 @@
+on:
+  schedule:
+  # Run quarterly - Jan, Mar, Jun, Sept, at 1PM UTC (~ 1AM NZT)
+  - cron: "0 13 * 1,3,6,9 *"
+jobs:
+  extract:
+    runs-on: ubuntu-latest
+    name: 'Extracts and prepares Freelex data for use by native applications'
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+      - uses: actions/checkout@v2
+      - run: make build update_assets
+      - run: tar -cf assets.tar.gz assets
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{steps.date.outputs.date}}
+          release_name: Quarterly release as at ${{steps.date.outputs.date}}
```

```suggestion
          tag_name: ${{ steps.date.outputs.date }}
          release_name: Quarterly release as at ${{ steps.date.outputs.date }}
```

🐘🐘🐘 just to be that guy 😅